### ISSUE TITLE
npm run scenario:share fails with Error: validator is not alive in v.0.1.0

### ISSUE THREAD

**Issue by sillysachin** [hyperledger/cactus#284]
**Describe the bug**
npm run scenario:share fails with following error 
[ERROR] Client - Error: validator is not alive

**To Reproduce**

Steps to reproduce the behavior on a successfully deployed Hyperledger Cactus cluster.

➜  simple-asset-transfer git:(d88f852) ✗ **npm run scenario:share**

NO Errors should have occured

**Logs/Stack traces**

➜  simple-asset-transfer git:(d88f852) ✗ npm run scenario:share

> @hyperledger-labs/blockchain-integration-framework.examples.simple-asset-transfer@0.1.0 scenario:share /Volumes/Workspace/BWork/cactus/examples/simple-asset-transfer
> cd scenarios && node share-pub-keys.js

[2020-09-16T20:09:02.553] [DEBUG] Client - tcp://127.0.0.1:7009 is not alive
[2020-09-16T20:09:02.556] [DEBUG] Client - tcp://127.0.0.1:7010 is not alive
[2020-09-16T20:09:02.557] [ERROR] Client - Error: validator is not alive
    at Function.askForPubKey (/Volumes/Workspace/BWork/cactus/examples/simple-asset-transfer/node_modules/@hyperledger-labs/blockchain-integration-framework/src/Client.js:88:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
[2020-09-16T20:09:02.558] [ERROR] Client - Error: validator is not alive
    at Function.askForPubKey (/Volumes/Workspace/BWork/cactus/examples/simple-asset-transfer/node_modules/@hyperledger-labs/blockchain-integration-framework/src/Client.js:88:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
[2020-09-16T20:09:02.558] [INFO] share-public-keys - Error: validator is not alive
    at Function.askForPubKey (/Volumes/Workspace/BWork/cactus/examples/simple-asset-transfer/node_modules/@hyperledger-labs/blockchain-integration-framework/src/Client.js:88:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @hyperledger-labs/blockchain-integration-framework.examples.simple-asset-transfer@0.1.0 scenario:share: `cd scenarios && node share-pub-keys.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @hyperledger-labs/blockchain-integration-framework.examples.simple-asset-transfer@0.1.0 scenario:share script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sachinsr/.npm/_logs/2020-09-16T14_39_02_575Z-debug.log
**Screenshots**

**Cloud provider or hardware configuration:**

Local Mac Laptop

**Operating system name, version, build:**

Use this command: `printf "$(uname -srm)\n$(cat /etc/os-release)\n"`

Darwin 19.6.0 x86_64

**Hyperledger Cactus release version or commit (git rev-parse --short HEAD):**

d88f852

**Hyperledger Cactus Plugins/Connectors Used**

 - Which DLT connectors are you using (Fabric, Quorum, Corda, Besu, etc.)
Fabric, Quorum, Corda

**Additional context**

**Comment by petermetz** [comment]
@sillysachin At the moment those old examples are broken unfortunately due to a backward incompatible breaking change with the Fabric image versioning (they deleted the latest tag IIRC). Please stay tuned for the updated examples that are coming soon!

**Comment by petermetz** [comment]
@sillysachin 

TLDR: Please track this for progress as the current issue is being superseded by it: https://github.com/hyperledger/cactus/issues/189

Because we started having issues with the old codebase related to OSS licensing as well, it was determined that it's just easier to retire it completely and go full throttle on the new examples that will demonstrate similar functionality but in a way that works much more consistently and is less resource intensive (hardware, especially RAM).
